### PR TITLE
shakey pidgeon

### PR DESCRIPTION
**Pull request by mcfiredrill**
closes https://github.com/datafruits/datafruits/issues/1123

https://github.com/datafruits/datafruits/assets/66243/94eabe0d-5603-4fd5-8346-38b0fc0a3a4d

### CODE REVIEW

**Comment by mcfiredrill** [comment]
Moved futsu to the corner here

**Comment by mcfiredrill** [comment]
I'd also like to add support to connect to the websocket to receive new notifs in real time, but for now it will just check on page refresh its probably good enough for a first version.

File: app/styles/shake.scss
Comment (dudethatbe) [inline app/styles/shake.scss:1]: cool keyframes 😎

File: types/glint.d.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { ComponentLike, HelperLike } from '@glint/template';
 declare module '@glint/environment-ember-loose/registry' {
   export default interface Registry {
     WelcomePage: ComponentLike;
+    Await: ComponentLike;
```
Comment (dudethatbe) [inline types/glint.d.ts:7]: huh whats this?

**Review by dudethatbe — APPROVED** [review]
seems good to me 👍 I noticed there's an entry for the pc-nav component, but not for sp-nav. not sure if that was intentional or not.

**Pull request merged**